data/AWSomeDayRoadshowIsrael.txt

```diff
@@ -0,0 +1,14 @@
+name:            AWSome Day Roadshow Israel
+url:             https://aws.amazon.com/events/awsomeday-roadshow-israel-2016/?mkt_tok=eyJpIjoiWlRVNFl6Qm1NV1UxWTJOaiIsInQiOiJ5djRhRFQ5K1dqUVhQWHdBVzVtcDJSWE52cTVoRTBBT1wvS2FGUDhQc2hKc1ZoMlBtWm5UNXRwTXU4N25ldDNyaUNaNjZqV3lEYUVQbVBcL1wvQ1ZRQWVnekdWSDQwTkd1dkZaXC9ZNzg2b1J2UHM9In0%3D
+start_date:      2016-11-21
+end_date:        2016-11-24
+cfp_date:        
+city:            Haifa | Jerusalem | Beer Sheva
```

Comment:
Multiple cities in one file is not supported. Let's have one of the cities and add a comment: fields with the other two.